code/script.js

```diff
@@ -229,36 +236,58 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
+   ('start function worked')
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+  //ADDED BY FAY 
+  generateBoard()
   // What else should happen when we start the game?
+  // ADDED setSecret to call next function
+  setSecret()
+  boardWrapper.classList.remove('board-wrapper-inactive')
+  winOrLose.classList.remove('win-or-lose-wrapper-active')
+ 
 }
-
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-
+  const value = questions.options[questions.selectedIndex].value
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
   // const value =
-
   currentQuestion = {
     category: category,
-    // value: value
+    value: value
   }
 }
 
 // This function should be invoked when you click on 'Find Out' button.
 const checkQuestion = () => {
+  // console.log(currentQuestion)
   const { category, value } = currentQuestion
-
+  // console.log('secret=',secret)
   // Compare the currentQuestion details with the secret person details in a different manner based on category (hair/eyes or accessories/others).
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
   if (category === 'hair' || category === 'eyes') {
+  
```

Comment:
This if statement is super clear and easy to read - I feel like easier to understand that what I did.